book/src/5.in-depth-guides/4.auth/1.quickstart.md

```diff
@@ -0,0 +1,74 @@
+# Quickstart
+
+In this section we will walk through creating an app that can access the [Microsoft Graph APIs](https://learn.microsoft.com/en-us/graph/overview) on behalf of the user by authenticating them with the [Microsoft Entra ID](https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-id) oauth provider. 
+
+> [!Note]
+> It is possible to authenticate the user into [other auth providers](https://learn.microsoft.com/en-us/azure/bot-service/bot-builder-concept-identity-providers?view=azure-bot-service-4.0&tabs=adv2%2Cga2#other-identity-providers) like Facebook, Github, Google, Dropbox, and so on.
+
+> [!Note]
+> This is an advanced guide. It is highly recommended that you are familiar with [creating an app](https://microsoft.github.io/teams-ai/2.getting-started/1.quickstart.html) and [running it in Teams](https://microsoft.github.io/teams-ai/2.getting-started/3.running-in-teams.html) before attempting to follow this guide.
+
+> [!Warning]
+> User authentication does not work with the developer tools setup. You have to run the app in Teams. Follow these [instructions](../../2.getting-started/3.running-in-teams.html#debugging-in-teams) to run your app in Teams.
+
+## Setup Instructions
+
+### Create an app with the `graph` template
+
+> [!tip]
+> Skip this step if you want to add the auth configurations to an existing app.
+
+Use your terminal to run the following command: 
+
+```sh
+teams new oauth-app --template graph
+```
+
+This command:
+1. Creates a new directory called `oauth-app`.
+2. Bootstraps the graph agent template files into it under `oauth-app/src`.
+3. Creates your agent's manifest files, including a `manifest.json` file and placeholder icons in the `oauth-app/appPackage` directory. The Teams [app manifest](https://learn.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema) is required for [sideloading](https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/deploy-and-publish/apps-upload) the app into Teams.
+
+### Add Teams toolkit auth configuration
+
+Open your terminal with the `oauth-app/` folder set as the curent working directory and run the following command:
```

Comment:
Move this after the cli command?

Reply:
the cli command should be run after these instructions